tasks/app/run.py

```diff
@@ -66,6 +67,11 @@ def warmup(
         #         skip_on_failure=True,
         #     )
 
+    if print_routes or app.debug:
+        log.info('Using route rules:')
+        for rule in app.url_map.iter_rules():
+            log.info('\t%r' % (rule,))
+
```

Comment:
see also `invoke app.endpoints.list`

Reply:
Yes, I found this specifically helpful when debugging routes.  I'd argue there is benefit in having both defined.  One is part of the debug printout and another is just a way to on-demand print the endpoints without starting a server.